src/components/login/index.js

```diff
@@ -1,12 +1,21 @@
 'use strict';
 
-exports.login = function(username){
+exports.login = function(username, password){
     exports.setUsername(username);
+    exports.setPassword(password);
     return exports.submit();
 };
 
+exports.loginDefault = function(){
+    return this.login('composer', 'composer');
```

Comment:
this.login is like exports.login, I would use exports.login since it's more nodejs like and you also used it in other places.